release/conf/tablet.flags.template

```diff
@@ -81,10 +81,10 @@
 #--key_entry_max_height=8
 
 # query conf
-# max table traverse iteration（full table scan/aggregation）,default: 50000
-#--max_traverse_cnt=50000
-# max table traverse pk number（batch query）, default: 5000
-#--max_traverse_pk_cnt=5000
+# max table traverse iteration(full table scan/aggregation),default: 0
+#--max_traverse_cnt=0
+# max table traverse unique key number(batch query), default: 0
+#--max_traverse_key_cnt=0
```

Comment:
must notice in changelog

Reply:
will it cause any issues if doing the upgrade? will the users keep their old config file?

Reply:
> will it cause any issues if doing the upgrade? will the users keep their old config file?

if using the old config file,  the result of online batch computing will be truncated as before

Reply:
> must notice in changelog

ok